content/reference/admin/private_locations/private_packages/index.md

```diff
@@ -9,71 +9,134 @@ weight: 22033
+      bindAddress = "0.0.0.0" # (optional, default: 0.0.0.0)
+      
+      # PKCS#12 certificate (optional)
+      certificate {
+        path = "/path/to/certificate.p12"
+        password = ${CERTIFICATE_PASSWORD} # (optional)
+      }
     }
   }
 }
 ```
 
 This configuration includes the following parameters:
-
-- **bucket**: The name of the bucket where packages are uploaded to on AWS S3.
-- **path:** The path of a folder in AWS S3. (optional)
 - **upload.directory**: This directory temporarily stores uploaded JAR files. (optional)
 - **server.port**: The port on which the control plane is listening for private package uploads.
 - **server.bindAddress**: The network interface to bind to. The default is `0.0.0.0`, which means all available network IPv4 interfaces.
+- **server.certificate**: The server P12 certificate for secure connection without SSL reverse proxy. (optional)
 
-After configuration, restart your control plane to start the server.
+### Control Plane repository
+
+#### AWS S3
+
+{{< alert warning >}}
+Control plane with private repository needs AWS permissions `s3:PutObject`, `s3:DeleteObject` and `s3:GetObject` on the bucket.
+{{< /alert >}}
+
+Once it is done, add the private repository configuration section in your [control plane configuration]({{< ref "../introduction" >}}) file:
+
+```bash
+control-plane {
+  repository {
+    # S3 Bucket configuration
+    bucket = "bucket-name"
+    path = "folder/to/upload" # (optional, default: root)
+  }
+}
+```
+
+This configuration includes the following parameters:
+- **bucket**: The name of the bucket where packages are uploaded to on AWS S3.
+- **path:** The path of a folder in AWS S3 bucket. (optional)
+
+#### GCP Cloud Storage
+
+{{< alert warning >}}
+Control plane with private repository needs GCP service account role with permissions `storage.objects.create`, 
+`storage.objects.delete` and `iam.serviceAccounts.signBlob` on the bucket.
+{{< /alert >}}
+
+```bash
+control-plane {
+  repository {
+    # S3 Bucket configuration
+    bucket = "bucket-name"
+    path = "folder/to/upload" # (optional, default: root)
+  }
+}
+```
+
+This configuration includes the following parameters:
+- **bucket**: The name of the bucket where packages are uploaded to on GCP Cloud Storage.
+- **path:** The path of a folder in Cloud Storage bucket. (optional)
 
 ## Usage
 
-You have to use Gatling plugins to create and upload your private packages. See Gatling documentation:
-* [Maven plugin](https://gatling.io/docs/gatling/reference/current/extensions/maven_plugin/)
-* [Gradle plugin](https://gatling.io/docs/gatling/reference/current/extensions/gradle_plugin/)
-* [SBT plugin](https://gatling.io/docs/gatling/reference/current/extensions/sbt_plugin/)
+After configuration, restart your control plane to start the server.
+
+### Creating a Private Package
+
+To create a private package, you can utilize Gatling Enterprise Cloud's public API. Follow these steps:
```

Comment:
I think we can mention that the easy way to create a private package is create a new simulation from the build tool.

Reply:
You can't...